go.mod

```diff
@@ -232,3 +232,5 @@ require (
 	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/minio/madmin-go/v2 => github.com/vadmeste/madmin-go/v2 v2.0.0-20230106172704-da31e43c554a
```

Comment:
madmin-go is merged, you can update the dependency.

Reply:
> madmin-go is merged, you can update the dependency.

Done